src/main/java/org/fogbowcloud/saps/engine/core/dto/TaskRequestDTO.java

```diff
@@ -10,14 +10,14 @@
      */
     private static final long serialVersionUID = 1L;
     
-    private String id;
+    private String label;
     private Map<String, String> requirements;
     private List<String> commands;
     private Map<String, String> metadata;
 
-    public TaskRequestDTO(String id, Map<String, String> requirements, List<String> commands,
+    public TaskRequestDTO(String label, Map<String, String> requirements, List<String> commands,
```

Comment:
this is non-trival. what is a `label`?. we should make the life of new devs easier.

Reply:
I created a problem (#75) to work on refactoring the code.